releng/org.eclipse.xtext.dev-bom/pom.xml

```diff
@@ -0,0 +1,326 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>org.eclipse.xtext</groupId>
+	<artifactId>xtext-dev.bom</artifactId>
+	<version>2.17.0-SNAPSHOT</version>
+	<packaging>pom</packaging>
+	<description>Xtext Third Party Dependencies (Bill of Materials)</description>
+
+	<properties>
+		<!-- ======== -->
+		<!-- Versions -->
+		<!-- ======== -->
+		<!-- 3rd party -->
+		<antlr.runtime-version>3.2</antlr.runtime-version>
+		<args4j-version>2.33</args4j-version>
+		<asm-version>7.0</asm-version>
+		<error_prone_annotations-version>2.0.15</error_prone_annotations-version>
+		<guava-version>[14.0,22.0]</guava-version>
```

Comment:
Is this correct? Should it be [14.0,22.0) ?

Reply:
@cdietrich

Reply:
21.0 instead?

Reply:
I think you are right. 22.0 should be excluded and usually we use the excluded version as upper bound. So `[14.0,22.0)` should be better here.